carousel-component/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+.idea
```

Comment:
.gitignore should be at the root directory, please merge one of the approved PRs so you don't have to add this file again and again